patches/chrome-browser-resources-settings-route.js.patch

```diff
@@ -27,11 +27,21 @@ index 337b8c13d19ca4ed3848057608f4a6de62dde82e..14dd9a6c8b348aedb0e482050d8c0c26
        r.SITE_SETTINGS_CAMERA = r.SITE_SETTINGS.createChild('camera');
 @@ -401,7 +407,9 @@ cr.define('settings', function() {
        }
- 
+
        r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
 +      // <if expr="_google_chrome">
        r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
 +      // </if>
        // <if expr="chromeos">
        r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
        // </if>
+@@ -419,6 +427,9 @@ cr.define('settings', function() {
+         r.TRIGGERED_RESET_DIALOG =
+             r.ADVANCED.createChild('/triggeredResetProfileSettings');
+         r.TRIGGERED_RESET_DIALOG.isNavigableDialog = true;
++        r.RESET_REWARDS_DIALOG =
```

Comment:
there is now a better way to do this stuff cc @petemill

Reply:
Is there documentation that you can link me to?